index.js

```diff
@@ -37,31 +57,53 @@ function validate(word) {
 }
 
 function stonksify(word) {
-  //If a word starts with a c replace it with a k
-  //Example: cat => kat
+  let modifiedCharacters = Array.apply(null, Array(word.length)).map(
+    function() {
+      return 0;
+    }
+  );
+
   word = word.toLowerCase();
-  if (word.indexOf("c") === 0) {
-    word = word.replace(/^c/, "k");
-    return word;
-  }
+  for (let i = 0; i < stonksifyRules.length; i++) {
+    let toReplace = new RegExp(stonksifyRules[i][0]);
+    let replaceWith = stonksifyRules[i][1];
+    let newWord = word.replace(toReplace, replaceWith);
+    console.log(
+      "Stonksify rule " +
+        i +
+        ": replace regexp '" +
+        toReplace +
+        "' with '" +
+        replaceWith +
+        "'."
+    );
```

Comment:
Can we use this 'template literals' to help make this more readable? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
done